src/qt/res/qml/AboutPane.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.9
+import QtQuick.Controls.Material 2.2
+
+Pane {
+    id: aboutPane
+
+    ColumnLayout {
+        id: aboutColumn
```

Comment:
I think we could use same convention used in c++, this could be `about_column`. Anyway, unused ids could be removed.